src/InstagramClone.Api/Controllers/UsersController.cs

```diff
@@ -130,6 +130,18 @@ public IActionResult GetUserFollowerById(Guid id)
             return Ok(listOfFollowedUsers);
         }
 
+        [HttpGet]
+        [Route("post/{id}")]
+        public IActionResult PostedItem(Guid id)
+        {
+            var post = _context.Posts.Find(id);
+
+            if(post == null)
+            {
+                return BadRequest();
+            }
+            return Ok(post.Content);
```

Comment:
the task says this functionality and the route should be `"api/posts/{id}"`. 

that means I should request this using a route like `http://localhost:3000:api/posts/<some post guid here>` but what you have done is different. 

Think about the concept of REST APIS and how each resource/item type should have its own separate route

Reply:
Ok. Done.